changed test to include actual access to api. (#15585)
demisto/content: Packs/CrowdStrikeMalquery/ReleaseNotes/1_0_8.md

#### Integrations
##### CrowdStrike Malquery

> - Added an api test call in addition to access token test when pressing `test` button.

Reviewer: ```suggestion
- Added an API test call when clicking the `test` button.
```
